transaction-status/src/parse_token.rs

```diff
@@ -41,641 +39,653 @@ pub fn parse_token(
             ));
         }
     }
-    match token_instruction {
-        TokenInstruction::InitializeMint {
-            decimals,
-            mint_authority,
-            freeze_authority,
-        } => {
-            check_num_token_accounts(&instruction.accounts, 2)?;
-            let mut value = json!({
-                "mint": account_keys[instruction.accounts[0] as usize].to_string(),
-                "decimals": decimals,
-                "mintAuthority": mint_authority.to_string(),
-                "rentSysvar": account_keys[instruction.accounts[1] as usize].to_string(),
-            });
-            let map = value.as_object_mut().unwrap();
-            if let COption::Some(freeze_authority) = freeze_authority {
-                map.insert(
-                    "freezeAuthority".to_string(),
-                    json!(freeze_authority.to_string()),
+    if let Ok(token_instruction) = TokenInstruction::unpack(&instruction.data) {
```

Comment:
Bah, this diff sucks, but all I did in this commit was move the `match instruction.accounts.iter().max()` bit up and then enclose the whole `match token_instruction` block within the following:

```rust
if let Ok(token_instruction) = TokenInstruction::unpack(&instruction.data) {
    /* ... */
} else {
    Err(ParseInstructionError::InstructionNotParsable(
        ParsableProgram::SplToken,
    ))
}
```

...then ran `rustfmt`.

Reply:
Ah thanks for the heads up, "hide whitespace" made this a delight :grin: